salt/addons/psp/manifests/10-podsecuritypolicy-privileged.yaml

```diff
@@ -0,0 +1,75 @@
+# The privileged PodSecurityPolicy is intended to be given
+# only to trusted workloads. It provides for as few restrictions as possible
+# and should only be assigned to highly trusted users.
+---
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: privileged
+  annotations: {}
```

Comment:
I believe you need to explicitly specify `seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'`. According to the docs, not including this annotation means the default cannot be changed.

Reply:
Updated to be explicit about this.